src/main/java/org/elasql/storage/tx/concurrency/ConservativeOrderedLockTable.java

```diff
@@ -370,20 +440,7 @@ void ixLock(Object obj, long txNum) {
 		}
 	}
 
-	/**
-	 * Releases the specified type of lock on an item holding by a transaction.
-	 * If a lock is the last lock on that block, then the waiting transactions
-	 * are notified.
-	 * 
-	 * @param obj
-	 *            a locked object
-	 * @param txNum
-	 *            a transaction number
-	 * @param lockType
-	 *            the type of lock
-	 */
-	void release(Object obj, long txNum, LockType lockType) {
-		Object anchor = getAnchor(obj);
+	void releaseLock(Object obj, long txNum, LockType lockType, Object anchor){

```

Comment:
It might be danger if we overload functions with identical parameter numbers.
Let's rename this function to `getLockersAndReleaseLock`